cmd/jaeger/internal/extension/jaegerstorage/extension_test.go

```diff
@@ -85,6 +88,9 @@ func TestStorageExtensionNameConflict(t *testing.T) {
 		Badger: map[string]badgerCfg.NamespaceConfig{
 			"foo": {},
 		},
+		Elasticsearch: map[string]esCfg.Configuration{
```

Comment:
not necessary, does not add anything to the test

Reply:
removed it!